pkg/model/firewall.go

```diff
@@ -86,6 +86,20 @@ func (b *FirewallModelBuilder) buildNodeRules(c *fi.ModelBuilderContext) error {
 		c.AddTask(t)
 	}
 
+	// Pods running in Nodes could need to reach pods in master/s
```

Comment:
What is this for? This is a tad open. Like we are opening etcd.  I know @liwenwu-amazon is planning on only running an operator on the master. Any other options for now?  If we need this we will need to print a warning message for the users.

Reply:
this is required to allow pod to pod communication (a pod in a node connecting to a pod in a master node)